### PR TITLE
Prevent one second initial delay

### PR DESCRIPTION
**Pull request by kornelijepetak**
Fixes:

* [x] Prevent one second initial delay

of #2040.

Now, the tab headers should show immediately, not only after a time service provides a current time.

🦑 at your earliest convenience.

### CODE REVIEW

File: Toggl.Giskard/Fragments/SelectTimeFragment.cs [hunk 1]
```diff
@@ -96,11 +96,28 @@ public override View OnCreateView(LayoutInflater inflater, ViewGroup container,
             onStopTimeChangedDisposable =
                 ViewModel.WeakSubscribe<PropertyChangedEventArgs>(nameof(ViewModel.StopTime), onStopTimeChanged);
 
+            initializeTabHeaders(ViewModel.StartingTabIndex);
```
Comment (willsb) [inline Toggl.Giskard/Fragments/SelectTimeFragment.cs]: Why is this needed even after the `StartWith`?

Comment (kornelijepetak) [reply Toggl.Giskard/Fragments/SelectTimeFragment.cs]: Because the `Initialize` method on the VM is called before `OnCreateView` here, so the weak subscription will not fire until there's a change in time after the `OnCreateView` has run. But before that, initial values have to be set up correctly.

File: Toggl.Foundation.MvvmCross/ViewModels/SelectTimeViewModel.cs [hunk 2]
```diff
@@ -251,6 +252,7 @@ public override void Prepare(SelectTimeParameters parameter)
             await base.Initialize();
 
             timeService.CurrentDateTimeObservable
```
Comment (willsb) [inline Toggl.Foundation.MvvmCross/ViewModels/SelectTimeViewModel.cs]: You should hold a reference to this observable

**Comment by kornelijepetak** [comment]
This PR must be postponed to after the #2085 is merged as the core things changed. This way, most of the requested changes will simply not be valid and this issue will be able to be merged.
